hoot-core-test/src/test/cpp/hoot/core/util/HttpTestServer.cpp

```diff
@@ -76,16 +76,23 @@ std::string HttpResponse::get_status_text()
   //  Convert the HTTP status code to status text
   switch (_status)
   {
-  case HttpResponseCode::HTTP_OK:                 return "OK";
-  case HttpResponseCode::HTTP_BAD_REQUEST:        return "Bad Request";
-  case HttpResponseCode::HTTP_NOT_FOUND:          return "Not Found";
-  case HttpResponseCode::HTTP_METHOD_NOT_ALLOWED: return "Method Not Allowed";
-  case HttpResponseCode::HTTP_CONFLICT:           return "Conflict";
-  default:                                        return "Unknown Status";
+  case HttpResponseCode::HTTP_OK:                     return "OK";
+  case HttpResponseCode::HTTP_BAD_REQUEST:            return "Bad Request";
+  case HttpResponseCode::HTTP_UNAUTHORIZED:           return "Unauthorized";
+  case HttpResponseCode::HTTP_NOT_FOUND:              return "Not Found";
+  case HttpResponseCode::HTTP_METHOD_NOT_ALLOWED:     return "Method Not Allowed";
+  case HttpResponseCode::HTTP_CONFLICT:               return "Conflict";
+  case HttpResponseCode::HTTP_GONE:                   return "Gone";
+  case HttpResponseCode::HTTP_PRECONDITION_FAILED:    return "Precondition Failed";
+  case HttpResponseCode::HTTP_INTERNAL_SERVER_ERROR:  return "Internal Server Error";
+  case HttpResponseCode::HTTP_BAD_GATEWAY:            return "Bad Gateway";
+  case HttpResponseCode::HTTP_SERVICE_UNAVAILABLE:    return "Service Unavailable";
+  case HttpResponseCode::HTTP_GATEWAY_TIMEOUT:        return "Gateway Timeout";
+  case HttpResponseCode::HTTP_BANDWIDTH_EXCEEDED:     return "Bandwidth Limit Exceeded";
+  default:                                            return "Unknown Status";
```

Comment:
Some unit test messages weren't being created properly without the text conversion to all HTTP response codes that we respond to in `OsmApiWriter`.